assets/capability/capabilities/alarm_co.json

```diff
@@ -105,7 +105,7 @@
           "no": "CO-alarmen er !{{på|av}}",
           "es": "La alarma de CO está !{{activada|desactivada}}",
           "da": "CO-alarmen er !{{on|off}}",
```

Comment:
Something else I noticed: I doubt that this is on/off in Danish, I think it should be `på|af`